tests/test_output.py

```diff
@@ -215,6 +215,18 @@ def test_get_lexer_not_found(self):
         assert get_lexer('xxx/yyy') is None
 
 
+@pytest.mark.parametrize("endpoint", [
+    "/encoding/utf8",
+    "/html",
+    "/json",
+    "/xml",
+])
+def test_ensure_contents_colored(httpbin, endpoint):
+    env = MockEnvironment(colors=256)
+    r = http('GET', httpbin + endpoint, env=env)
+    assert COLOR in r
```

Comment:
You need to ensure headers are not included in the output for the `COLOR in r` check to be meaningful (e.g., [add `--body`](https://github.com/httpie/httpie/pull/1244#discussion_r769772969))

Reply:
Right, makes sense.